source/docs/software/advanced-controls/geometry/coordinate-systems.rst

```diff
@@ -10,7 +10,7 @@ The field coordinate system (or global coordinate system) is an absolute coordin
 
 .. image:: diagrams/coordinate-systems-1.drawio.svg
 
-Below is an example of a field coordinate system overlayed on the 2020 FRC field.
+Below is an example of a field coordinate system overlayed on the 2020 FRC field. The red axes shown are for the red alliance, and the blue axes shown are for the blue alliance.
```

Comment:
Sorry to keep changing it but I keep thinking of things. Maybe the coordinates should be located on the alliance wall with the outline of a robot starting position?

Reply:
I think having the coordinates at the alliance station wall would imply that the origin is at the alliance station wall.